Practice2/src/styles/components/common/card.css

```diff
@@ -0,0 +1,45 @@
+.card {
+    border-radius: 8px;
```

Comment:
Please double-check the tab size in this file and all the same cases.

Reply:
Fixed